EIPS/eip-draft_tx_expir.md

```diff
@@ -0,0 +1,39 @@
+---
+eip: <to be assigned>
+title: Transaction Expiration
+description: Proposes a Transaction Expiration Block for every transaction.
+author: Lucas Vinzon<lucasvinzon@gmail.com>
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category (*only required for Standards Track): Core
+created: 2022-01-25
+requires (*optional):
+---
+
+## Abstract
+When in Ethereum Mempool, every transaction should expire after a certain amount of blocks resulting in no cost to its users. 15.000 blocks would effectively have every transaction on the Mempool for two days.
+
+## Motivation
+Gas cost, usability and serving its purpose for its users. Prevents unexperienced users to waste resources.
+
+## Specification
+Once in the Mempool ALL transactions MUST expire after a certain amount of blocks. It is OPTIONAL to lower the amount of blocks it will be live on the Mempool. It SHALL NOT result in gas cost to users.
```

Comment:
The specification section should include more technical details about the specific changes that need to happen.

For example, are you suggesting that the transaction type be modified to include an expiry block? If so, where?

Another important question to consider is whether this applies to all transactions, or if you're introducing a new transaction type (see [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718).)

Reply:
Hey Sam! Thanks for your response. I'll have to apologize since most of what I'm discussing I'm not fully aware of how to implement. I'd just like to open a conversation and allow other devs to bring in suggestions on how this could end up being implemented.
Would it make more sense to introduce a new transaction type that becomes adopted as a standard? It would be ideal if this applied to all transactions instead.

Reply:
If you just want to discuss the idea, I recommend just creating a thread on ethresear.ch or ethereum-magicians.  Alternatively, you could bring it up in the Ethereum R&D Discord server.  Generally, you want to create an EIP once you have something to specify/standardize rather than just as a focal point for conversation.